### PR TITLE
Add Crystopia.net

### PR DESCRIPTION
**Pull request by tnsjesper**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by tnsjesper** [comment]
What is wrong?

**Comment by DoJapHD** [comment]
Seems to have been a bug, works now

File: minecraft_servers/crystopia/manifest.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "server_name": "crystopia",
+    "nice_name": "Crystopia",
+    "direct_ip": "crystopia.net",
+    "supported_languages": [
+       "en",
+       "de"
+    ],
+    "social": {
+      "web": "https://www.crystopia.net",
+      "web_shop": "https://shop.crystopia.net",
```
Comment (DoJapHD) [inline minecraft_servers/crystopia/manifest.json:11]: Not reachable

File: minecraft_servers/crystopia/manifest.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      "web_support": "https://www.crystopia.net/index.php?route=/wiki",
```
Comment (DoJapHD) [inline minecraft_servers/crystopia/manifest.json:12]: 404

File: minecraft_servers/crystopia/manifest.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      "twitter": "crystopianet",
+      "discord": "https://discord.com/invite/q7xW3MH425",
+      "youtube": "https://www.youtube.com/@crystopia"
+    },
+    "gamemodes": {
+      "smp": {
+        "name": "SMP", 
+        "color": "#FFA500",
+        "url": "https://www.crystopia.net/index.php?route=/games/smp/",
```
Comment (DoJapHD) [inline minecraft_servers/crystopia/manifest.json:21]: 404

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by tnsjesper** [comment]
Yes i have fixed.

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**